packages/backEnd/src/services/adds.services.ts

```diff
@@ -22,11 +22,6 @@ const s3 = new S3({
   },
 });
 
-interface MulterRequest extends Request {
-  files: any;
-  file: any;
-}
-
 const uploadedFilesToSpaces = async (requsetFiles) => {
```

Comment:
We have upload files in src/tools/image.ts. I think we should refactor updateFile method to be reusable in all places where we need. Method should take two parameters - files and allowed mimetype.
Using mimetype, we will be able to determine what type of file the user will be able to upload in a given place.

Reply:
I that good point to have reusable function for uploading images.